Packs/CyberArkIdentity/Integrations/CyberArkIdentityEventCollector/CyberArkIdentityEventCollector_description.md

```diff
@@ -5,16 +5,22 @@ Before you start, you need to get the tenant ID, app ID, username, and password.
 ## Configuration Parameters
 
 **Server URL**    
-The endpoint to get the logs. For example, ``https://{{tenant}}.my.idaptive.app/``.
+The endpoint to get the logs. For example, ``https://{{tenant}}.my.idaptive.app/``
```

Comment:
I would change this line to:
The CyberArk Identity URL to get the logs from. For example, ``https://{{tenant}}.my.idaptive.app/``